rir/src/compiler/translations/pir_2_rir/liveness.cpp

```diff
@@ -9,6 +9,7 @@ namespace rir {
 namespace pir {
 
 LivenessIntervals::LivenessIntervals(unsigned bbsSize, CFG const& cfg) {
+    maxLive = 0;
```

Comment:
Where is this being used?

Reply:
admittedly not in this pr. do you mind?

Reply:
Sorry, but I'd prefer having this change in the commit that uses it (otherwise it looks like dead code). This one is small enough that it shouldn't be too much trouble... (famous last words)